tests/e2e/autoscaling/00-assert.yaml

```diff
@@ -11,5 +11,5 @@ status:
   conditions:
     - type: Available
       status: "True"
-  cpus: 1
+  cpus: "1"
```

Comment:
We shouldn't need to use a string here, I think? Curious why that's required if we do.

Reply:
This is the "canonical form" for Kubernetes. Even if we pass 1, it will work but the created resource will have "1". This assert checks Kubernetes output (not input), so we need quotes

Reply:
Resolving, now that it's been added